src/app/work-item/work-item-board/work-item-board.component.html

```diff
@@ -35,7 +38,7 @@
                     </div>
                   </div>
                   <div [dragula]='"first-bag"' class="card-wrapper">
-                    <div class="board-card card-pf card-pf-view card-pf-view-select card-pf-view-single-select" [class.active]="isSelected(item)" *ngFor='let item of lane.workItems' routerLink="./detail/{{item.id}}" [attr.data-id]="item.id"
+                    <div class="board-card flex-container in-column-direction flex-grow-1" *ngFor='let item of lane.workItems' [attr.data-id]="item.id"
```

Comment:
@mindreeper2420 here you go - https://github.com/fabric8io/fabric8-planner/pull/1692 let me know if you find anything else.